node-tests/asset-manifest-generator-test.js

```diff
@@ -62,6 +62,20 @@ describe('asset-manifest-generator', function() {
       return verifyAssetManifest('resources', { prepend: '/resources/' });
     });
 
+    it('can ignore specific files', function() {
+      return verifyAssetManifest('full-minus-ignored-files', {
+        prepend: '/bundles',
+        filesToIgnore: [ 'blog/assets/engine.css', 'chat/assets/engine.js' ]
```

Comment:
Minor quibble about the naming, we have a similar concept in broccoli-funnel land and call it `exclude`. Thoughts?

Reply:
I originally had this as `ignore`, but changed it because there may be a use case in the future where we want an entire "bundle" ignored, in which case we may want some added specificity. Also, I prefer the explicitness. That said, I don't feel super strongly about it either way.